audio_separator/separator/uvr_lib_v5/demucs/hdemucs_rocm.py

```
@@ -0,0 +1,820 @@
+        self,
+        chin,
+        chout,
+        kernel_size=8,
+        stride=4,
+        norm_groups=1,
+        empty=False,
+        freq=True,
+        dconv=True,
+        norm=True,
+        context=0,
+        dconv_kw={},
```

Ruff (0.15.6)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)